src/CodeViewerDemo/CodeExamples.h

```diff
@@ -23,12 +23,21 @@ inline const QString testing_example =
   kTreeSize == 3;
   _Tree_;
 
+  // Class members
+  Track* track_;
+  __int__;
+  m_Text_y;
+  Track::SortTracks();
+  class TrackManager {
+
   // Function unit tests
   int my_function(void)
+  int MyFunction(void)
```

Comment:
should this end with a `;`? Similar above.

Reply:
I will polish a little bit CodeExamples to make code which have some sense.